source/includes/templates/tabs-gs.rst

```diff
@@ -16,17 +16,17 @@
    .. raw:: html
 
       <div class="tabs">
-        <ul class="nav nav-tabs nav-justified">
+        <ul tab-strip tab-strip--singleton" role="tablist">
```

Comment:
I am confused about the bem structure implemented here.  I think it'd be something like:

- block: tabs

- elements: tabs__tab-strip, tabs__content, tabs__panel( Maybe? I see what you're doing though by adding the ID to the class so maybe disregard this one)

- modifiers: tabs__tab-strip--singleton